documentation/cspell.yaml

```diff
@@ -0,0 +1,19 @@
+version: '0.2'
```

Comment:
Documentation in the repo should probably be spell checked. This is just for the detections in `ci-fix.md`